python/fabnonacci.py

```diff
@@ -0,0 +1,16 @@
+def fb(n):
+
+    a=0 # a is the first number in fabonacci  series.
+    b=1 # b  is second number in fabonacci series .
+    print(a)
+    print(b)
+    """
+    by printing the above condition the fabonacci series is : 0 1 1 and so till n 
+    """
+    for i in range(2,n+1): # by using for it will iterate till n
```

Comment:
Using iteration & switch values are nice idea for printing out fabonacci series. 👍

Reply:
very veryy thankk  u sir